packages/react-cosmos-apollo-proxy/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "react-cosmos-apollo-proxy",
+  "version": "2.0.0-beta.23",
```

Comment:
I think you need to make this `2.0.0-beta.28` for the tests to pass. The example references 2.0.0-beta.28 so it tries to download it from npm instead of linking from the monorepo.

Reply:
Oh, yes! Thanks for the heads up!